plugins/obs-ffmpeg/ffmpeg-mux/ffmpeg-mux.c

```diff
@@ -871,6 +871,7 @@ static int ffmpeg_mux_write_av_buffer(void *opaque, uint8_t *buf, int buf_size)
 	return buf_size;
 }
 
+static bool ffmpeg_mux_is_network(struct ffmpeg_mux *ffm);
```

Comment:
Personally I would move the `ffmpeg_mux_is_network` function above `open_output_file` rather than using a forward declaration, since it's already close to this code.

Reply:
done